pkgs/tools/security/web-eid-app/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib
+, mkDerivation
+, fetchFromGitHub
+, cmake
+, gtest
+, pcsclite
+, pkg-config
+, qmake
+, qttranslations
+}:
+
+mkDerivation rec {
+  pname = "web-eid-app";
+  version = "2.2.0";
+
+  src = fetchFromGitHub {
+    owner = "web-eid";
+    repo = "web-eid-app";
+    rev = "v${version}";
+    sha256 = "sha256-TOzOcPY4m7OdCfaAXyc/joIHe2O2YbyDrXiNytPMKSk=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    cmake
+    pkg-config
+  ];
+
+  buildInputs = [
+    gtest
```

Comment:
Shouldn't that be a checkInput?

Reply:
Actually, looking further I don't even see a reference to gtest in the code anymore, only in upstream's CI configuration. Could you double check if this is even needed?

Reply:
This is a dependency by `lib/libelectronic-id/lib/libpcsc-cpp/CMakeLists.txt:68`. If I remove it, I get the following error:

```
CMake Error at /nix/store/9d67jcm1wj09wqgffh8v5mkgil9psnrw-cmake-3.25.3/share/cmake-3.25/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find GTest (missing: GTEST_LIBRARY GTEST_INCLUDE_DIR
  GTEST_MAIN_LIBRARY)
Call Stack (most recent call first):
  /nix/store/9d67jcm1wj09wqgffh8v5mkgil9psnrw-cmake-3.25.3/share/cmake-3.25/Modules/FindPackageHandleStandardArgs.cmake:600 (_FPHSA_FAILURE_MESSAGE)
  /nix/store/9d67jcm1wj09wqgffh8v5mkgil9psnrw-cmake-3.25.3/share/cmake-3.25/Modules/FindGTest.cmake:270 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  lib/libelectronic-id/lib/libpcsc-cpp/CMakeLists.txt:68 (find_package)

```

Reply:
Ah, thanks for the pointer -- GitHub code search doesn't recurse through submodules apparently...

Reply:
I added a small comment, in case someone stumbled over this again.